Podfile

```diff
@@ -48,12 +50,12 @@ pod 'ARAnalytics/HockeyApp'
 
 pod 'CardFlight'
 pod 'ECPhoneNumberFormatter'
-pod 'UIImageViewAligned', :git => "https://github.com/orta/UIImageViewAligned.git"
+pod 'UIImageViewAligned', :git => "https://github.com/ashfurrow/UIImageViewAligned.git"
```

Comment:
My PR for this got merged, and it's on trunk, not sure what you added but removing this may be doable